### ISSUE TITLE
Can we train it on GPU?

### ISSUE THREAD

**Issue by mitesh741** [otenim/Xception#3]

**Comment by otenim** [comment]
@miteshgithub 

`fine_tune.py` and `inference.py` are automatically executed using gpus if your machine has more than one gpu and the backend library `tensorflow-gpu' is correctly installed. Please refer to https://www.tensorflow.org/install/gpu for details.

**Comment by mitesh741** [comment]
Thanks a bunch! One more question. I trained two models: one with 3 class and high resolution images 1080p, model trained with good accuracy. But when I trained another model with 10 classes with less than or equal to 720p then I am not getting accuracy. I run more than 25 epochs with same lr as in previous model. Is image resolution a problem?
